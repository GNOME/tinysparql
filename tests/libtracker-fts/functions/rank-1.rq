SELECT ?u fts:rank(?u) { ?u fts:match 'bananas' } order by desc fts:rank(?u)
