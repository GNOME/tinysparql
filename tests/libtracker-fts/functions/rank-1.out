"http://www.example.org/test#1"	"20.0"
"http://www.example.org/test#2"	"20.0"
"http://www.example.org/test#4"	"20.0"
"http://www.example.org/test#3"	"0.0"
